src/tracks/InstrumentTrack.cpp

```diff
@@ -1148,6 +1174,29 @@ class fxLineLcdSpinBox : public LcdSpinBox
 			gui->fxMixerView()->setFocus();// set focus to fxMixer window
 			//engine::getFxMixerView()->raise();
 		}
+
+		virtual void contextMenuEvent( QContextMenuEvent* event )
+		{
+			// for the case, the user clicked right while pressing left mouse-
+			// button, the context-menu appears while mouse-cursor is still hidden
+			// and it isn't shown again until user does something which causes
+			// an QApplication::restoreOverrideCursor()-call...
+			mouseReleaseEvent( NULL );
+
+			QPointer<CaptionMenu> contextMenu = new CaptionMenu( model()->displayName(), this );
+
+			// This condition is here just as a safety check, fxLineLcdSpinBox is aways
+			// created inside a TabWidget inside an InstrumentTrackWindow
+			if ( InstrumentTrackWindow* window = dynamic_cast<InstrumentTrackWindow*>( (QWidget *)this->parent()->parent() ) )
```

Comment:
@tresf My question was more about the `parent()->parent()` thing, but thanks for the concern :-)